sql/api/src/main/scala/org/apache/spark/sql/streaming/StatefulProcessor.scala

```diff
@@ -28,7 +28,7 @@ import org.apache.spark.sql.errors.ExecutionErrors
  */
 @Experimental
 @Evolving
```

Comment:
Can we add `since`

Reply:
The API is still evolving and under active development. Thats why we have kept it as `package private` till we decide which Spark release it will become public in